pkgs/development/python-modules/userpath/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, click
+, distro
+}:
+
+buildPythonPackage rec {
+  pname = "userpath";
+  version = "1.4.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256="0mfjmvx286z1dmnrc7bm65x8gj8qrmkcyagl0vf5ywfq0bm48591";
+  };
+
+  propagatedBuildInputs = [ click distro ];
+
+  doCheck = false;
```

Comment:
it's still odd to see tests disabled, please give a comment as to why:

```
  # test suite is difficult to emulate in sandbox due to shell manipulation
```

Reply:
Added.